### PR TITLE
Check that the tutorials build with v0.9-dev of PennyLane

### PR DESCRIPTION
**Pull request by josh146**

### CODE REVIEW

**Comment by josh146** [comment]
We should also update the requirements.txt file as each of the plugins are finalized

**Comment by josh146** [comment]
<h3>Website build</h3>
<strong>Commit:</strong> e95c9ef01e619bf67e76782d7914658a72c4e103

<strong>Circle build number:</strong> 671

<strong>Website zip:</strong> <a href=https://671-214003948-gh.circle-artifacts.com/0/tmp/qml_html.zip>https://671-214003948-gh.circle-artifacts.com/0/tmp/qml_html.zip</a>

<strong>Website url:</strong> <a href=https://671-214003948-gh.circle-artifacts.com/0/_build/html/index.html>https://671-214003948-gh.circle-artifacts.com/0/_build/html/index.html</a>

<em>Please double check the rendered website build to make sure everything is correct.</em>

File: requirements.txt [hunk 1]
```diff
@@ -10,12 +10,12 @@ sympy==1.4
 pillow==6.2.0
 pip==19.2
 protobuf==3.8.0
-pennylane==0.8.0
+git+https://github.com/XanaduAI/pennylane.git@#egg=pennylane
```
Comment (antalszava) [inline requirements.txt:13]: The PennyLane version error seems to arise from the fact that this is using an `egg` whereas the Cirq and Qiskit plugin links will build a wheel. Due to this fact, they run first (for some weird reason), setting the PL version to `0.8`. Making the PennyLane build a wheel as well should make sure that it is indeed installed prior to any other plugin.

`Requirement already satisfied: pennylane from git+https://github.com/XanaduAI/pennylane.git@#egg=pennylane in ./venv/lib/python3.7/site-packages (from -r requirements.txt (line 13)) (0.8.0)`

Locally this change made the PL installation to be executed first
```suggestion
git+https://github.com/XanaduAI/pennylane.git
```

Comment (josh146) [reply requirements.txt:13]: Good catch! Let's give this a go and see if it works.

File: requirements.txt [hunk 2]
```diff
@@ -2,27 +2,27 @@
...
-pennylane-cirq==0.8.0
-pennylane-qiskit==0.8.0
-pyquil==2.17
-scipy==1.3.2
+git+https://github.com/XanaduAI/pennylane.git
+git+https://github.com/XanaduAI/pennylane.git#subdirectory=qchem
+git+https://github.com/XanaduAI/pennylane-sf.git
+git+https://github.com/rigetti/pennylane-forest.git
+git+https://github.com/XanaduAI/pennylane-cirq.git
+git+https://github.com/XanaduAI/pennylane-qiskit.git
```
Comment (josh146) [inline requirements.txt:18]: Before this is merged:

* Wait for PL-core and plugins to be released

* Revert this back to the (pinned) PyPI latest release

* Wait for the PR build to pass

* Merge it in

**Pull request merged**